samigo/samigo-app/src/java/com/corejsf/UploadRenderer.java

```diff
@@ -80,15 +81,16 @@ public void decode(FacesContext context, UIComponent component){
     ExternalContext external = context.getExternalContext();
     HttpServletRequest request = (HttpServletRequest) external.getRequest();
     String clientId = component.getClientId(context);
-    FileItem item = (FileItem) request.getAttribute(clientId+UPLOAD);
-    // check if file > maxSize allowed
+    WrappedUpload item = wrapUpload(request, clientId + UPLOAD);
+
     log.debug("clientId ="+ clientId);
-    log.debug("fileItem ="+ item);
+    log.debug("wrappedUpload ="+ item);
```

Comment:
Use the following to avoid the evaluation of item:
`log.debug("wrappedUpload = {}", item);`